demo/helper.py

```diff
@@ -117,11 +117,15 @@ def has_include_string(s):
 def copy_lib_files(output_path):
```

Comment:
Maybe documentation to describe the helper function?  e.g.:
```
def copy_lib_files(output_path):
        """
        summary of what helper does.

        Returns:
        type(e.g. int): description of returned value 
        """
```

Reply:
ok